web/public/templates/dojos/my-dojos.dust

```diff
@@ -50,4 +10,13 @@
       </div>
     </div>
   </div>
+  <cd-expanding-dojo-card ng-repeat="dojo in myDojos" dojo="dojo" user="currentUser" start-expanded="{{ myDojos.length === 1 }}">
```

Comment:
When you say sub-state of my-dojos, what do do you mean...?
It expands if there is at least 1 dojo but closes if more than 1. Yes???